ui-tests/src/test/java/io/syndesis/qe/steps/connections/wizard/ConnectionWizardSteps.java

```diff
@@ -0,0 +1,41 @@
+package io.syndesis.qe.steps.connections.wizard;
+
+import static org.junit.Assert.assertThat;
+
+import static org.hamcrest.Matchers.is;
+
+import static com.codeborne.selenide.Selenide.$;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import cucumber.api.java.en.And;
+import io.syndesis.qe.instances.SharedInstances;
+import io.syndesis.qe.pages.connections.wizard.ConnectionWizard;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ConnectionWizardSteps {
+
+    @Autowired
+    private ConnectionWizard wizard;
+
+    @And("^(\\w+) navigates to the next Connection wizard step \"([^\"]*)\"$")
```

Comment:
`@When` or `@Then`. Imho this is when step.

Reply:
I think it doesn't matter at all. You can use any of @When, @Then, @And in a scenario regardless what annotation has been used.
I can change it but as long as we don't have rules for that... If you still think I should change it I'll do it. Should I?

Reply:
There's best practice how to handle `when, then, and`. In a nutshell when triggers action, then verify result, and is a substitution for a previous when or then. Any other rule needed? We may not enforce it too much, but...

Reply:
ok, I'm changing this one to When